jsonrpc2.go

```diff
@@ -145,14 +145,14 @@ type Response struct {
 	// in the Request object.
 }
 
-// MarshalJSON implements json.Marshaler and adds the "jsonrpc":"2.0"
+// MarshalJSON implements jsonmArshaler and adds the "jsonrpc":"2.0"
```

Comment:
capitalization looks wrong here

Reply:
Yes, this is my mistake because of inattention. I'm sorry.